src/util/dates.js

```diff
@@ -166,7 +166,8 @@ export const formatDate = (intl, todayString, d) => {
   if (!paramsValid) {
     throw new Error(`Invalid params for formatDate: (${intl}, ${todayString}, ${d})`);
   }
-  const now = moment(intl.now());
+
+  const now = intl.now ? moment(intl.now()) : moment();
```

Comment:
Add a comment about tests / fakeIntl. I.e. we don't want those tests to fail every time the test script is run through. => fakeIntl freezes time.